cvprac/cvp_api.py

```diff
@@ -96,6 +96,26 @@ def get_cvp_info(self):
             self.clnt.set_version(data['version'])
         return data
 
+    def update_username(self, username, password, role):
```

Comment:
The function name update_username seems a bit misleading. Isn't this function technically updating a given user accounts password and role? It isn't actually changing the username itself is it? If my assumption is right maybe the function name of update_user, or update_user_info, or update_user_data?

Reply:
yes you're right, we could go for `update_user` which is close to the name of the API (`updateUser.do`) or `update_user_info` ?

Reply:
I like update_user as it keeps the name shorter and closer to the API